src/pfe/file-watcher/server/src/controllers/projectStatusController.ts

```diff
@@ -67,6 +68,7 @@ class ProjectState {
     appImageLastBuild?: string;
     buildImageLastBuild?: string;
     detailedAppStatus?: DetailedAppStatus;
+    notificationID?: string;
```

Comment:
It looks like `notificationID` is defined in both `ProjectState` and `DetailedAppStatus`. I think we can remove it from `ProjectState`.